pkg/controller.v1/tensorflow/job_test.go

```diff
@@ -663,6 +663,18 @@ var _ = Describe("Test for controller.v1/common", func() {
 			wantTFJobIsRemoved: false,
 			wantErr:            false,
 		}),
+		Entry("No error with completionTime is nil if suspended", &cleanUpCases{
+			tfJob: tftestutil.NewTFJobWithCleanupJobDelay(1, 2, 0, nil),
+			runPolicy: &kubeflowv1.RunPolicy{
+				TTLSecondsAfterFinished: nil,
```

Comment:
```suggestion
				TTLSecondsAfterFinished: ptr.To[int32](10),
```

Shouldn't we need to specify the ttlSecondsAfterFinished?
Because previously, there were bugs in the situations where the Job has ttlsSecondsAfterFinished and has been suspended, right?

Reply:
hmm yes, I this should be another test case